### DISCUSSION TITLE
It is possible that the option to connect with Hedera Chains may not be available when creating a new WalletConnect project.

### DISCUSSION THREAD

**Discussion by fnLog0** [WalletConnect/walletconnect-monorepo#2599] [Developer Support]
I need assistance with enabling support for Hedera Chains in the "chains" field when creating new projects. Additionally, I would like to know if it's possible to use a project ID before publishing it.

**Comment by glitch-txs** [comment]
Hey, Hedera is currently not supported but you can ask for this following the docs https://docs.walletconnect.com/2.0/advanced/multichain/overview#register-chain-with-the-explorer

Yes, you can use your project ID right away once is created
